### PR TITLE
Change RealmID to 1

### PR DESCRIPTION
**Pull request by Err0r1**

### CODE REVIEW

**Comment by mdjong1** [comment]
Why though?

**Comment by OMGhixD-OG** [comment]
> Why though?
> 
> Shouldn't this table just be auto-increment primary key on `id`?

You do make a point, it usually just pulls whatever data is available in the realmlist table. There shouldn't be a need to specify realmid in this file.

**Comment by angrytoilet** [comment]
For some reason this was causing an error when i was working with Err0r on my situation.

**Comment by angrytoilet** [comment]
> > Why though?
> > Shouldn't this table just be auto-increment primary key on `id`?
> 
> You do make a point, it usually just pulls whatever data is available in the realmlist table. There shouldn't be a need to specify realmid in this file.

For some reason this was causing an error when i was working with Err0r on my situation.

**Comment by OMGhixD-OG** [comment]
> > > Why though?
> > > Shouldn't this table just be auto-increment primary key on `id`?
> > 
> > 
> > You do make a point, it usually just pulls whatever data is available in the realmlist table. There shouldn't be a need to specify realmid in this file.
> 
> For some reason this was causing an error when i was working with Err0r on my situation.

with or without this commit?

**Comment by angrytoilet** [comment]
> > > > Why though?
> > > > Shouldn't this table just be auto-increment primary key on `id`?
> > > 
> > > 
> > > You do make a point, it usually just pulls whatever data is available in the realmlist table. There shouldn't be a need to specify realmid in this file.
> > 
> > 
> > For some reason this was causing an error when i was working with Err0r on my situation.
> 
> with or without this commit?

removing the "?" and changing it to 1 resolved the issue.  For some reason, the "?" wasnt finding next.

**Comment by OMGhixD-OG** [comment]
Fair enough, but how do we suppose to include realmid 2 if that is ever added?

**Comment by Err0r1** [comment]
> Fair enough, but how do we suppose to include realmid 2 if that is ever added?

auto_increment is set to ID 2
so it should automatically have ID 2 when a new realm is added

**Comment by mdjong1** [comment]
Okay my brain decided to turn on; you should'nt specify the ID if it's auto-increment, i.e.:

```php
$query = ("INSERT INTO realms(hostname`, `username`, `password`, `char_database`, `world_database`, `cap`, `realmName`, `console_username`, `console_password`, `console_port`, `emulator`, `realm_port`, `override_port_world`, `override_username_world`, `override_password_world`, `override_hostname_world`, `override_port_char`, `override_username_char`, `override_password_char`, `override_hostname_char`)
                        VALUES('".$hostname."',
                            '".$username."',
                            '".$password."',
                            '".$characters."',
                            '".$world."',
                            '".$cap."',
                            '".$realmName."',
                            '".$console_username."',
                            '".$console_password."',
                            '".$console_port."',
                            '".$emulator."',
                            '".$realm_port."',
                            '".$db_port."',
                            '".$username."',
                            '".$password."',
                            '".$hostname."',
                            '".$db_port."',
                            '".$username."',
                            '".$password."',
                            '".$hostname."');");
```

File: install/do_realms.php [hunk 1]
```diff
@@ -47,9 +47,9 @@ public function __construct()
                 echo json_encode(array("success" => false, "message" => "Looks like your world database doesn't exist"));
                 exit();
             }
-			
+
             $query = ("INSERT INTO realms(`id`, `hostname`, `username`, `password`, `char_database`, `world_database`, `cap`, `realmName`, `console_username`, `console_password`, `console_port`, `emulator`, `realm_port`, `override_port_world`, `override_username_world`, `override_password_world`, `override_hostname_world`, `override_port_char`, `override_username_char`, `override_password_char`, `override_hostname_char`)
```
Comment (mdjong1) [inline install/do_realms.php:51]:
```suggestion
            $query = ("INSERT INTO realms(`hostname`, `username`, `password`, `char_database`, `world_database`, `cap`, `realmName`, `console_username`, `console_password`, `console_port`, `emulator`, `realm_port`, `override_port_world`, `override_username_world`, `override_password_world`, `override_hostname_world`, `override_port_char`, `override_username_char`, `override_password_char`, `override_hostname_char`)
```

File: install/do_realms.php [hunk 2]
```diff
@@ -47,9 +47,9 @@ public function __construct()
...
-                        VALUES('?',
+                        VALUES('1',
```
Comment (mdjong1) [inline install/do_realms.php:52]:
```suggestion
                        VALUES(
```

**Comment by mdjong1** [comment]
Obsolete by https://github.com/FusionGen/FusionGEN/pull/112

**Pull request closed without merging**